### PR TITLE
creating a package

### PR DESCRIPTION
**Pull request by elizavetasemenova**
- divided content into Julia-beginners and those who want to write packages
- for beginners a new page called "Contributing is for everyone"
- for those who want to create packages, page called "Creating a Julia package"

### CODE REVIEW

File: contribute/creating_package.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+A detailed page on Julia packages is available [here](http://pkgdocs.julialang.org/v1/). Including the part on [creating packages](http://pkgdocs.julialang.org/v1/creating-packages/).
+
+Julia packages consist of modules. Hence, before creating a package, make sure to understand how [modules](https://docs.julialang.org/en/v1/manual/modules/) work in Julia.
+
+
+Here we provide a step-by-step example, using [PkgTemplates.jl](https://juliahub.com/docs/PkgTemplates/IGiQL/0.6.4/), inspired by the [video](https://www.youtube.com/watch?v=QVmU29rCjaA&t=112s) of Chris Rackauckas.
+
+In case of any issues, remember that [help](https://github.com/elizavetasemenova/www.julialang.org/blob/main/contribute/contributing_for_everyone.md#where-to-get-help) is available.
+
+## What you need before you start
```
Comment (lilianabs) [inline contribute/creating_package.md:12]: We can rephrase it as Prerequisites :)

File: contribute/creating_package.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+- an [IDE](https://github.com/elizavetasemenova/www.julialang.org/blob/main/contribute/contributing_for_everyone.md#what-are-the-ide-options)
+- a GitHub account
+
+## Starting a package
+
+- create a folder to store your package. We use Atom as an Editor. We created a folder called 'tiddlywinks.jl'. In Atom, we open the GitHub panel, and opt to 'initialize and publish on GitHub'. Now we can go to our GitHub page and see that 'tiddlywinks.jl' has appeared there, as well as locally.
```
Comment (lilianabs) [inline contribute/creating_package.md:19]: This section is a bit confusing :( We could reprhase it as follows:

1. You need to create a folder to store a new package. For example, if you use Atom as an IDE, create a new folder called `tiddlywinks.jl`. Then, open the GitHub panel and select **Initialize and publish on GitHub**. 
  Now, you can go to your GitHub page and see that 'tiddlywinks.jl' has appeared there, as well as locally.
2. In your terminal application, use the change directory (cd) command to change the current working directory to the location of your package.
3. In the Julia REPL, install `PkgTemplate.jl` as follows:
   
   ```
    using Pkg
    Pkg.add("PkgTemplates")
   ```

File: contribute/creating_package.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+- inside this folder, in the Julia REPL install PkgTemplate.jl with
+```
+using Pkg
+Pkg.add("PkgTemplates")
+```
+
+## Creating package template
+
+Now `PkgTemplates` has been installed and we want to use it to create a template for a package.
+```
```
Comment (lilianabs) [inline contribute/creating_package.md:30]: We need to specify where to put this code :) Is it inside tiddlywinks.jl?

File: contribute/creating_package.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+               Codecov(),
+               Coveralls(),
+               AppVeyor(),
+           ],
+       )
+
+t("tiddlywinks")
+```
+
+As we have not specified the folder, the package has been created at `~/.julia/dev` on Linux, and at `username/julia/dev` in Windows. If this is not the repository we want to be working from, let's move the files to the original repo which we have created. Now in `tiddlywinks.jl` we have
```
Comment (lilianabs) [inline contribute/creating_package.md:48]: In technical documentation, we usually use second person (you) to address our readers :) I suggest rewriting this paragraph to be consistent with the above ones. See this as reference: https://developers.google.com/style/person

File: contribute/creating_package.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+using tiddlywinks
+using Test
+
+@testset "tiddlywinks.jl" begin
+    @test tiddlywinks.tiddly_greet() == "tiddlywinks is working"
+    @test tiddlywinks.tiddly_greet() != "hello world"
+end
+```
+
+## Running tests
```
Comment (lilianabs) [inline contribute/creating_package.md:82]: I'd suggest adding an introductory line such as "To run tests on your package:"

**Review by lilianabs — COMMENTED** [review]
This is a great document! I left some comments and suggestions regarding technical writing good practices :) I'll follow the steps on how to create a new package.

File: contribute/creating_package.md (hunk 4 above)
Comment (elizavetasemenova) [reply contribute/creating_package.md:58]: really cool document to go over, thanks

**Pull request merged**